branch_tags/views.py

```diff
@@ -1,3 +1,18 @@
-from django.shortcuts import render
+import json
+
+from django.views import View
+from django.http  import JsonResponse
+
+from .models import UserTag
+
+class UserTagListView(View) :
+    def get(self, request) :
+        userTags = UserTag.objects.all()
```

Comment:
Python naming convention 지켜주세요~
```suggestion
        user_tags = UserTag.objects.all()
```